src/config.py

```diff
@@ -27,7 +27,7 @@ class KafkaConfig:
 
     def __init__(self, charm):
         self.charm = charm
-        self.default_config_path = f"{self.charm.config['data-dir']}/config"
+        self.default_config_path = f"{DATA_DIR}/config"
```

Comment:
should `/config` be appended to `DATA_DIR` literal directly?

Reply:
This is what was done before. I would keep it like this for the moment.